java-client/src/main/java/org/opensearch/client/transport/aws/AwsSdk2Transport.java

```diff
@@ -85,6 +87,7 @@ public class AwsSdk2Transport implements OpenSearchTransport {
     private final String signingServiceName;
     private final Region signingRegion;
     private final JsonpMapper defaultMapper;
+    @Nonnull
```

Comment:
No particular take on that, but I believe all this fields are should not be null.